package/kedro_viz/api/rest/requests.py

```diff
@@ -7,5 +7,6 @@ class DeployerConfiguration(BaseModel):
     """Credentials for Deployers."""
 
     platform: str
+    preview: bool
```

Comment:
lets keep the naming consistent with cli...something like `is_preview_enabled` or `is_preview_enabled_for_all_nodes` if we want this to be more detailed.

Reply:
Done.